docs/instances.md

```diff
@@ -4,10 +4,13 @@
 
 This page provides a list of `Websurfx` instances provided by us and our community. 
 
-|URL|Network|Version|Location|Behind Cloudflare?|Maintained By|TLS|IPv6|Comment|
+|URL|Network|Version|Location|Behind Cloudflare?|Status|TLS|IPv6|Comment|
 |-|-|-|-|-|-|-|-|-|
-|https://websurfx.co/|www|edge|🇺🇸 US|||✅|❌||
-|https://websurfx.onrender.com/|www|edge|🇺🇸 US|||✅|❌||
-|https://alamin655-websurfx.hf.space/|www|v0.21.4|🇺🇸 US||[websurfx project](https://github.com/neon-mmd/websurfx)|✅|❌||
+|https://websurfx.co|www|rolling|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/2/status"></a>|✅|❌||
+|https://alamin655-spacex.hf.space|www|rolling|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/5/status"></a>|✅|❌||
+|https://websurfx.instance.pp.ua|www|rolling|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/7/status"></a>|✅|✅||
+|https://alamin655-websurfx.hf.space|www|stable|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/6/status"></a>|✅|❌||
+
+Check the Uptime of the Instances - https://status.websurfx.pp.ua
```

Comment:
```suggestion
```

I think I would suggest removing this because this link has already been mentioned in the instances table. So I feel it looks repetitive. :slightly_smiling_face: